2018-11-30T11:07:22|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\ScopeFoundry\logged_quantity.py", line 241, in update_value
    self.hardware_set_func(self.val)
  File "C:\LabPrograms\Python\Camera-ScopeFoundry\CameraMeasurement.py", line 248, in setRecord
    self.settings.save_h5.disable()
AttributeError: 'LoggedQuantity' object has no attribute 'disable'

2018-11-30T11:07:23|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\ScopeFoundry\logged_quantity.py", line 241, in update_value
    self.hardware_set_func(self.val)
  File "C:\LabPrograms\Python\Camera-ScopeFoundry\CameraMeasurement.py", line 242, in setSaveH5
    self.settings.record.setEnabled()
AttributeError: 'LoggedQuantity' object has no attribute 'setEnabled'

